packages/material-ui/src/Modal/Modal.test.js

```diff
@@ -518,10 +524,14 @@ describe('<Modal />', () => {
 
   describe('two modal at the same time', () => {
     it('should open and close', () => {
+      // Using a Fade component here to run checks after the Modal has mounted and unmounted
+
       const TestCase = props => (
         <React.Fragment>
           <Modal open={props.open}>
-            <div>Hello</div>
```

Comment:
Can we keep the old test and add a new one?

Reply:
I’ll give it a go, before the test had a race condition but I’ve changed things since then